### ISSUE TITLE
Video logging missing

### ISSUE THREAD

**Issue by vwxyzjn** [wandb/client#1091]
`wandb --version && python --version && uname` 
* Weights and Biases version: 0.9.0
* Python version: 3.7
* Operating System: Linux

### Description
For some reasons, the videos of my experiments are somehow not logged correctly. Please see the following link as an example, where are the file and code section of the panel contains nothing.

https://app.wandb.ai/cleanrl/cleanrl.benchmark/runs/2wioc6zu/files?workspace=user-costa-huang

![image](https://user-images.githubusercontent.com/5555347/84148748-6a2ff700-aa2d-11ea-8b4a-f1fb129f8e49.png)

In fact, it seems the videos and files of this particular run are somehow logged to another run. As an example, the following link contains and abnormal amount of files associated with the run

https://app.wandb.ai/cleanrl/cleanrl.benchmark/runs/27wq5kno/files/media/videos

![image](https://user-images.githubusercontent.com/5555347/84148789-761bb900-aa2d-11ea-858b-cbe464a11d59.png)

My expectation is that the runs should contain a reasonable number of videos as shown in the following link

https://app.wandb.ai/cleanrl/cleanrl.benchmark/runs/1lej2r8u/files/media/videos

![image](https://user-images.githubusercontent.com/5555347/84148828-89c71f80-aa2d-11ea-9c05-6832833c3dff.png)

### What I Did

I ran the following file multiple times using various parameters

https://app.wandb.ai/cleanrl/cleanrl.benchmark/runs/1ddwhlvh/code?workspace=user-costa-huang

The actual command to run is 

```
docker run -d --shm-size="500m" --cpuset-cpus="0" -e WANDB=xxxxxxxxxxxxxxxxxxxxxxxx vwxyzjn/cleanrl_shared_memory:latest python ppo_atari_visual.py --gym-id BeamRiderNoFrameskip-v4 --total-timesteps 10000000 --wandb-project-name cleanrl.benchmark --wandb-entity cleanrl --prod-mode --capture-video --seed 1
docker run -d --shm-size="500m" --cpuset-cpus="1" -e WANDB=xxxxxxxxxxxxxxxxxxxxxxxx vwxyzjn/cleanrl_shared_memory:latest python ppo_atari_visual.py --gym-id BeamRiderNoFrameskip-v4 --total-timesteps 10000000 --wandb-project-name cleanrl.benchmark --wandb-entity cleanrl --prod-mode --capture-video --seed 2
docker run -d --shm-size="500m" --cpuset-cpus="2" -e WANDB=xxxxxxxxxxxxxxxxxxxxxxxx vwxyzjn/cleanrl_shared_memory:latest python ppo_atari_visual.py --gym-id QbertNoFrameskip-v4 --total-timesteps 10000000 --wandb-project-name cleanrl.benchmark --wandb-entity cleanrl --prod-mode --capture-video --seed 1
docker run -d --shm-size="500m" --cpuset-cpus="3" -e WANDB=xxxxxxxxxxxxxxxxxxxxxxxx vwxyzjn/cleanrl_shared_memory:latest python ppo_atari_visual.py --gym-id QbertNoFrameskip-v4 --total-timesteps 10000000 --wandb-project-name cleanrl.benchmark --wandb-entity cleanrl --prod-mode --capture-video --seed 2
docker run -d --shm-size="500m" --cpuset-cpus="4" -e WANDB=xxxxxxxxxxxxxxxxxxxxxxxx vwxyzjn/cleanrl_shared_memory:latest python ppo_atari_visual.py --gym-id SpaceInvadersNoFrameskip-v4 --total-timesteps 10000000 --wandb-project-name cleanrl.benchmark --wandb-entity cleanrl --prod-mode --capture-video --seed 1
docker run -d --shm-size="500m" --cpuset-cpus="5" -e WANDB=xxxxxxxxxxxxxxxxxxxxxxxx vwxyzjn/cleanrl_shared_memory:latest python ppo_atari_visual.py --gym-id SpaceInvadersNoFrameskip-v4 --total-timesteps 10000000 --wandb-project-name cleanrl.benchmark --wandb-entity cleanrl --prod-mode --capture-video --seed 2
docker run -d --shm-size="500m" --cpuset-cpus="6" -e WANDB=xxxxxxxxxxxxxxxxxxxxxxxx vwxyzjn/cleanrl_shared_memory:latest python ppo_atari_visual.py --gym-id PongNoFrameskip-v4 --total-timesteps 10000000 --wandb-project-name cleanrl.benchmark --wandb-entity cleanrl --prod-mode --capture-video --seed 1
docker run -d --shm-size="500m" --cpuset-cpus="7" -e WANDB=xxxxxxxxxxxxxxxxxxxxxxxx vwxyzjn/cleanrl_shared_memory:latest python ppo_atari_visual.py --gym-id PongNoFrameskip-v4 --total-timesteps 10000000 --wandb-project-name cleanrl.benchmark --wandb-entity cleanrl --prod-mode --capture-video --seed 2
docker run -d --shm-size="500m" --cpuset-cpus="8" -e WANDB=xxxxxxxxxxxxxxxxxxxxxxxx vwxyzjn/cleanrl_shared_memory:latest python ppo_atari_visual.py --gym-id BreakoutNoFrameskip-v4 --total-timesteps 10000000 --wandb-project-name cleanrl.benchmark --wandb-entity cleanrl --prod-mode --capture-video --seed 1
docker run -d --shm-size="500m" --cpuset-cpus="9" -e WANDB=xxxxxxxxxxxxxxxxxxxxxxxx vwxyzjn/cleanrl_shared_memory:latest python ppo_atari_visual.py --gym-id BreakoutNoFrameskip-v4 --total-timesteps 10000000 --wandb-project-name cleanrl.benchmark --wandb-entity cleanrl --prod-mode --capture-video --seed 2
```

However, it is also possible to run without a docker container such as the following

```
for seed in {1..2}
do
    (sleep 0.3 && nohup xvfb-run -a python ppo_atari_visual.py \
    --gym-id BeamRiderNoFrameskip-v4 \
    --total-timesteps 10000000 \
    --wandb-project-name cleanrl.benchmark \
    --wandb-entity cleanrl \
    --prod-mode \
    --capture-video \
    --seed $seed
    ) >& /dev/null &
done

for seed in {1..2}
do
    (sleep 0.3 && nohup xvfb-run -a python ppo_atari_visual.py \
    --gym-id QbertNoFrameskip-v4 \
    --total-timesteps 10000000 \
    --wandb-project-name cleanrl.benchmark \
    --wandb-entity cleanrl \
    --prod-mode \
    --capture-video \
    --seed $seed
    ) >& /dev/null &
done

for seed in {1..2}
do
    (sleep 0.3 && nohup xvfb-run -a python ppo_atari_visual.py \
    --gym-id SpaceInvadersNoFrameskip-v4 \
    --total-timesteps 10000000 \
    --wandb-project-name cleanrl.benchmark \
    --wandb-entity cleanrl \
    --prod-mode \
    --capture-video \
    --seed $seed
    ) >& /dev/null &
done

for seed in {1..2}
do
    (sleep 0.3 && nohup xvfb-run -a python ppo_atari_visual.py \
    --gym-id PongNoFrameskip-v4 \
    --total-timesteps 10000000 \
    --wandb-project-name cleanrl.benchmark \
    --wandb-entity cleanrl \
    --prod-mode \
    --capture-video \
    --seed $seed
    ) >& /dev/null &
done

for seed in {1..2}
do
    (sleep 0.3 && nohup xvfb-run -a python ppo_atari_visual.py \
    --gym-id BreakoutNoFrameskip-v4 \
    --total-timesteps 10000000 \
    --wandb-project-name cleanrl.benchmark \
    --wandb-entity cleanrl \
    --prod-mode \
    --capture-video \
    --seed $seed
    ) >& /dev/null &
done
```

**Comment by vanpelt** [comment]
@vwxyzjn we're looking into the root cause, it's likely something that regressed with the latest release.  You can downgrade to 0.8.36 with `pip install wandb==0.8.36`.  We'll update here once we've root caused the issue.

**Comment by vanpelt** [comment]
@vwxyzjn we just released 0.9.1 of the library with fixes for this issue.  Thanks for reporting and sorry for the inconvenience.  The bug resulted in all files being uploaded to the newest run when many runs were executing in parallel.
